LayoutKit/Views/BatchUpdates.swift

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import Foundation
+
+
+/// A set of updates to apply to a `ReloadableView`.
+public struct BatchUpdates {
+    public var insertItems = [NSIndexPath]()
```

Comment:
My preference is weak, but I think present tense makes the most sense here.

This object represents a request to a reloadable view to apply a set of updates. This object is passed to the reloadable view concurrently with the new state to apply, so it doesn't feel like the updates have already happened (i.e. no past tense).

If the code was structured such that the state got updated first and then the reloadable view had to react (e.g. by handling an event callback) then past tense would make more sense.